docs/core/equations.qmd

```diff
@@ -149,19 +149,21 @@ plt.show()
 The precipitation term is given by
 
 $$
-    Q_P = P \cdot A(u).
+    Q_P = P \cdot A
 $$ {#eq-precip}
 
-Here $P = P(t)$ is the precipitation rate and $A$ is the wetted area. $A$ is a
-function of the storage $u = u(t)$: as the volume of water changes, the area of the free water
-surface may change as well, depending on the slopes of the surface waters.
+Here $P = P(t)$ is the precipitation rate and $A$ is the maximum area given in the `Basin / profile` table.
```

Comment:
Is it really the maximum area that is used? Considering 

```julia
fixed_area = basin.area[i][end]
```
it depends on the ordering of these vectors.

Reply:
Indeed it depends on the ordering, but that is enforced to be increasing. We rely on these orderings in many places in the code, so I don't think an explicit max is needed.

https://deltares.github.io/Ribasim/core/usage.html#basin-profile